data/human/human missions.txt

```diff
@@ -6890,3 +6890,109 @@ mission "Hogshead's Dilemma II"
+			choice
+				`	(Show the butler Caesar, and ask to see his owner.)`
+
+			`	The butler's frown deepens, and then sighs. "Card, please." He holds out a silver tray and awaits your calling card.`
+			choice
+				`	(Explain you don't have a calling card.)`
+					goto nocard
+				`	(Place your card on the tray.)`
+
+			`	The butler's eyebrows raise in the slightest hint of surprise when you place your card on the tray. He promptly turns and disappears into the overlarge house. You, Caesar, and the footman all wait for half an hour before a woman arrives at the front entrance, wearing an exceedingly complicated gown.`
+				goto boolinbrook
+
+			label nocard
+			`	The butler smiles smugly. "How unfortunate. In that case I have no choice but to bring you through the servant's entrance. It's only proper."`
+			`	Instead of taking you up the wide stairs to the beautifully inscribed wooden doors, he takes you around the side to a small plain door. He leaves you there with the footman to wait.`
+			`	A quarter of an hour later a woman wearing an exceedingly impractical dress arrives.`
+
+			label boolinbrook
+			`	"Mr. Boolinbrook says you've brought back my darling See-See?" From down her nose she looks at you and waits for your response.`
+			choice
+				`	"Your dog is safe and sound with me."`
+				`	"I found Caesar alone out on Calda, by the spaceport."`
+				`	"Here he is, you're welcome."`
+
+			`	Before you can respond, Caesar starts yapping happily and runs up to the lady.`
+			`	She snatches up the dog and draws it to her bosom. "Oh my darling See-See! It has been most dreadful waiting for news of you!"`
+			`	After a moment she turns her attention back to you, looking slightly softer than before. "See-See tires of the parks here, and so every Tuesday I send her to Calda for a change of pace. It's good for her constitution, you know. Anyways, the horrid man who normally ferries our dogs on these little excursions had some sort of episode. A stroke, or heart-attack, you get the idea. He had an assistant return the dogs, but in the confusion they forgot my little See-See. I was sure some gauche tourist would have snatched her up and eaten her by now."`
+			`	She turns her attention back to the dog, and speaks in a baby-voice, "But, instead, this gallant captain saved you!"`
+			`	The lady snaps her fingers, and the butler is immediately at her side. "We must make this person whole. I can't be owing favors to vagabonds. Imagine the talk!`
+			`	She procures a credit chip from somewhere in her gown and passes it to you. "I realize this is probably a fortune to you. I recommend you find a financial advisor and invest it well."`
+			`	With that she turns and leaves; the butler likewise disappears, and you're escorted by the footman back to your vehicle.`
+			`	You look down at the credit chip in your hand and read its value; 1,000 credits.`
```

Comment:
I commend you on what you have written. I don't have the ability to write stories but I can spot when things don't feel right. 

This whole section feels confusing
-  the butler doesn't recognize the dog but later we are told that butler informed 'the lady of the manor' that you had returned her dog
- the waiting 'time' is too long for somebody that was very upset about her "darling See-See" 
- choices
  - "Your dog is safe and sound with me."     Can't she see the dog?
  - "I found Caesar alone out on Calda, by the spaceport."     No, you where in a park a considerable distance walk away from the spaceport.
  - You offer choices but then start the next dialogue with "Before you can respond" 
- "See-See tires of the parks here, ...." Doesn't the dog live on a huge estate that takes at least 45 minutes to reach the mansion?
- "But, instead, this **_gallant captain_** saved you!"  but in the next breath she calls you a vagabond and treats you with zero respect.

Reply:
Partly, these are explained by the fact that rich Hestians (or rich Paradise Worlders) seem quite irrational to the player; they live on a different plane of existence to the player character (no matter how rich the player character is). They don't trust outsiders; they have these romanticised or incorrect notions about who ship captains are, how much money they actually make at all, and what you actually did. A lot of their demeanour is very much extraneous and shallow; taking 15 minutes to pick up a dog they supposedly love? Maybe they don't love the dog. Maybe that's why they send the dog away to Calda every Tuesday. They have this huge estate with hundreds of buildings, but they might not necessarily use them, or allow people and dogs to go roaming around, for whatever reason.

> "I found Caesar alone out on Calda, by the spaceport." No, you where in a park a considerable distance walk away from the spaceport.

Fair point though.

> You offer choices but then start the next dialogue with "Before you can respond"

It's not unprecedented, you're put in a similar situation in the Free Worlds campaign. (The more practical reason... it's sometimes easier to put the player in this situation than to write all of that.)

Reply:
Bene has the right of it above.
The disconnect is that this person is absurdly wealthy. It takes 45 minutes to get to their home because they live on a 100,000 acre practical fiefdom. As for it being a ranch, it's quite common for very wealthy people to own ranches, or even hobby farms they don't work on, or do very little on. Maybe they 'train' the horses, or work on 'dressage' for the Olympics. Those same people will of course also have servants. 

Same goes for her imagining her dog needs to be taken to a different planet because her 'humble yard' bores him. Not all wealthy people behave this way, but many certainly do. Dog's getting rides in private jets etc, taken to private beaches. 

As for the gallant captain vs vagabond phrasing this has the same reasoning as above. The same way a wealthy person could say they could never live without their beloved maid/butler/nurse/aid etc but then also be dismissive of them as human beings, or at least other people like them (poors? immigrants?). She's inconsistent because she doesn't view or understand you as a real person.